src/components/button/base/Button.tsx

```diff
@@ -0,0 +1,30 @@
+type Icon = {
+  leftIcon?: JSX.Element
+  rightIcon?: JSX.Element
+}
+
+type ButtonProps = {
+  url?: string
+  name: string
+  icon?: Icon
+}
+
+const Button: React.FC<ButtonProps> = ({ url, name, icon }): JSX.Element => (
```

Comment:
I said that a `Link` without an `url` was more a `Button`, but here you say that an `anchor` tag element (or a `link`) is always a `Button`, whatever the props.

Be careful with that, because declaring a `Button` component as a generic React Component which only renders a `a` tag is a non-sense,  IMHO.

How this will work if we want to use this component as a regular button?